core/src/smartcontracts/isi/tx.rs

```diff
@@ -3,15 +3,16 @@
 use eyre::{Result, WrapErr};
 use iroha_data_model::{
     prelude::*,
-    query::error::{FindError, QueryExecutionFailure},
+    query::error::{FindError, QueryExecutionFailure as Error},
```

Comment:
Why not using `QueryExecutionFailure` as-is? Would make more sense than just `Error`, taking into account that this error is not exclusive here (there is also `FindError`)